agent/bench-scripts/tests/pbench-trafficgen/test-22.post

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-${_real_cat} ${_testdir}/trafficgen_test-22_1970.01.01T00.00.00/result.json >> ${_testout}
+${_real_cat} ${_testdir}/trafficgen_test-22_1900.01.01T00.00.00/result.json >> ${_testout}
```

Comment:
Since you changed `agent/bench-scripts/tests/pbench-trafficgen/test-22.opts` to use `_rundir`, should you be making the analogous change here?  (Or, is that variable not accessible in this context?)

Reply:
Sure, I've moved that change from the next commit back to this one.